tests/unit/test__init__.py

```diff
@@ -140,29 +124,16 @@ def tearDown(self):
     async def test_acquire_lock_already_locked(self):
         mock_pool = patch.object(self.pg_driver, '_pool').start()
         mock_con = Mock_CM()
-        mock_con.fetchval.return_value = futurized(True)
+        mock_con.fetchval.return_value = futurized(False)
```

Comment:
It looks odd, how this change makes the test valid (according to the name)

Reply:
The logic was moved from python method to sql procedure. Method returns the same, so assert was not changed.
`self.assertFalse(await self.pg_driver._acquire_lock('mock_preset_name'))`